jobfair-frontend/src/containers/CompanyRegisterForm/CompanyRegisterForm.container.jsx

```diff
@@ -45,29 +46,21 @@ const CompanyRegisterFormContainer = () => {
 
   const handleOnFinish = async () => {
     try {
-      const res = await handleRegisterCompany(form.getFieldsValue(true));
-      const responseManager = await handleRegisterCompanyManager(form.getFieldsValue(true), res.data.id);
-      if (res.status === 200 && responseManager.status === 201) {
-        Modal.success({
-          title: 'Register company manager successfully !',
-          width: '30rem',
-          closable: true,
-          maskClosable: true,
-          okText: 'OK',
-          keyboard: false,
-          onOk: () => history.push(PATH.LOGIN_PAGE),
-          content: <RegisterSuccessContentComponent email={form.getFieldValue('email')} />
-        });
-      }
+      await handleRegisterCompany(form.getFieldsValue(true));
+      await handleRegisterCompanyManager(form.getFieldsValue(true), res.data.id);
+      Modal.success({
```

Comment:
if the request fail the Modal.success still show up

Reply:
if request is failed, each handleRegisterCompany and handleRegisterCompanyManager will catch errors and show error modals